.go-version

```diff
@@ -0,0 +1 @@
+1.21.7
```

Comment:
Per https://github.com/coredns/coredns/issues/6454#issuecomment-1979466900, we may use 1.21.8 for CVE update.

Reply:
> Per [#6454 (comment)](https://github.com/coredns/coredns/issues/6454#issuecomment-1979466900), we may use 1.21.8 for CVE update.

@pacoxu just bumped to 1.21.8 in a new commit